src/app/navmeshWorker.js

```diff
@@ -0,0 +1,389 @@
+/* eslint-env worker */
```

Comment:
This worker file is pretty impressive - nice work! Could you add a comment near the top with a short description/example of how it is used, for future project contributors with less web worker experience?